components/staticFeeds/sample_notifications.json

```diff
@@ -1,35 +0,0 @@
-{"notifications" :
```

Comment:
:question: Is the removal of this file intended?

Reply:
Yes to all the above. They are no longer used. 

@vertein There shouldn't be any downstream consequences, since apps using older frame versions depend on their overlay projects to point to the correct feeds, yes?

Reply:
Correct.  Apps were either overriding or using sample notifications.  Either way, if you're on this version of the product, you don't need this file. 

I didn't check yet, but if these were all in a single commit like 'Remove unneeded notification files', that would be great. ;)